frontend/src/routes/explore/+page.svelte

```diff
@@ -1,6 +1,10 @@
 <script lang="ts">
     import Watchlist from "$components/Watchlist.svelte";
     import News from "$components/News.svelte";
+    import Graph from "$components/Graph.svelte";
+
+    let w = 0;
```

Comment:
`w` and `h` aren't very descriptive, would be better to have `graphWidth` and `graphHeight`.